### DISCUSSION TITLE
Highlight.io

### DISCUSSION THREAD

**Discussion by nahtnam** [coollabsio/coolify#2577] [Service Template Requests]
Highlight.io is a Sentry alternative. They have a simpler "hobby" deployment: https://www.highlight.io/docs/getting-started/self-host/self-hosted-hobby-guide

The compose isn't too complicated: https://github.com/highlight/highlight/blob/main/docker/compose.hobby.yml

**Comment by tconroy** [comment]
Just wanted to +1 this 

in the meantime: trying to figure out how to manually deploy this on Coolify 😅 there's some .sh scripts you need to run to deploy so not as straight forward as repo clone -> run docker-compose

**Reply by underthestars-zhy** [reply]
did u figure out?

**Reply by dilincoln** [reply]
> did u figure out?

up

**Reply by tconroy** [reply]
I did not, switched to sentry.

**Reply by underthestars-zhy** [reply]
how do u host sentry on coolify

**Comment by atilladeniz** [comment]
someone has this yaml?

**Comment by atilladeniz** [comment]
I'm building the template right now here https://github.com/atilladeniz/highlight-coolify

https://github.com/atilladeniz/highlight-coolify/blob/main/docker/docker-compose-coolify.yaml

**Comment by Eggtron** [comment]
Something new? I am still struggeling with hosting it on coolify

**Comment by biolimbo** [comment]
I started work on a template [here](https://github.com/biolimbo/coolify-highlight-service-template), it is **work in proggress** not functional yet. Still have some errors with kafka and postgres. I will work on debugging it over the weekend. Opened to suggestions and contributions.
